tests/rkt_os_arch_test.go

```diff
@@ -161,6 +161,12 @@ func TestMissingOrInvalidOSArchFetchRun(t *testing.T) {
 	defer osArchTestRemoveImages(tests)
 
 	for i, tt := range tests {
+		// Skip this test as it will always fail due to a missing hash in the
+		// checked output.
```

Comment:
I suspect it was broken by https://github.com/coreos/rkt/commit/594bd1ca60de6bae06d70aaa7c8fe04c281283e5#commitcomment-15066199.

The function `getMissingOrInvalidTests` looks very repetitive. There should be a way to refactor it with more [Data-driven testing](https://en.wikipedia.org/wiki/Data-driven_testing).

/cc @krnowak